linux/scripts/start-web.py

```diff
@@ -7,11 +7,13 @@
 from pantos.servicenode.configuration import config
 from pantos.servicenode.configuration import load_config
 
+print('Loading the configuration...')
 load_config()
+print('Configuration loaded')
 
 USER_NAME = 'pantos-service-node'
 APP_DIRECTORY = '/opt/pantos/pantos-service-node'
-WSGI_FILE = f'{APP_DIRECTORY}/wsgi.py'
+WSGI_FILE = f'{APP_DIRECTORY}/lib/python3.10/site-packages/pantos/servicenode/wsgi.py'
```

Comment:
The python version should not be hardcoded right?

Reply:
We control the python version right now, this version is hardcoded in debian/rules itself. We would need to do a find in this file in order to find out which directory uses this file.